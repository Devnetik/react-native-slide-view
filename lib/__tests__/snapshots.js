import {Text} from 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import SlideView from '../slideView.js';

it('renders correctly without props', () => {
  const tree = renderer.create(<SlideView/>).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly with children', () => {
  const tree = renderer.create(<SlideView><Text>This is a slideview</Text></SlideView>).toJSON();
  expect(tree).toMatchSnapshot();
});
