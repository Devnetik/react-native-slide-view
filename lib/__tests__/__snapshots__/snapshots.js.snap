exports[`test renders correctly with children 1`] = `<View />`;

exports[`test renders correctly without props 1`] = `<View />`;
